ios/UniversalAnalyticsPlugin.m

```diff
@@ -100,6 +100,23 @@ - (void) setAnonymizeIp: (CDVInvokedUrlCommand*)command
   [self.commandDelegate sendPluginResult:pluginResult callbackId:command.callbackId];
 }
 
+- (void) setOptOut: (CDVInvokedUrlCommand*)command
+{
+  CDVPluginResult* pluginResult = nil;
+  NSString* optout = [command.arguments objectAtIndex:0];
+
+  if ( ! _trackerStarted) {
+    pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_ERROR messageAsString:@"Tracker not started"];
+    [self.commandDelegate sendPluginResult:pluginResult callbackId:command.callbackId];
+    return;
+  }
+
+  [[GAI sharedInstance] setOptOut:optout];
```

Comment:
Please convert to boolean the pointer, I think you can do it this way:
[[GAI sharedInstance] setOptOut:[optout:boolValue]];